src/chart-addons/stacked-legend.unit.spec.js

```diff
@@ -64,7 +64,14 @@ describe("Stacked Legend", () => {
         open: true,
         domain: [0, 100],
         position: "bottom-left",
-        range: ["#27aeef", "#ea5545"]
+        range: [
```

Comment:
restored previous test, and it runs green.

NOTE: more tests need to be added to account for the new versioning. no time for that now. more of a TODO